packages/react/src/QuestionnaireBuilder/QuestionnaireBuilder.tsx

```diff
@@ -189,6 +190,13 @@ function ItemBuilder<T extends Questionnaire | QuestionnaireItem>(props: ItemBui
     } as T);
   }
 
+  function handleRepeatableValue(item: QuestionnaireItem): void {
```

Comment:
Would a better name for this function be `toggleRepeatable`?  Because it looks like it's used for both "make repeatable" and "remove repeatable"?